packages/toolpad-app/src/toolpad/propertyControls/GridColumns.tsx

```diff
@@ -176,6 +188,13 @@ function GridColumnsPropEditor({
           <React.Fragment>
             <DialogTitle>Edit columns</DialogTitle>
             <DialogContent>
+              <IconButton
+                aria-label="Recreate columns"
```

Comment:
We should probably add a tooltip as well.

@prakhargupta1 @gerdadesign nominating this component as well for a redesign. Perhaps we could even put this inline in the inspector, instead of in a dialog?

Reply:
good idea, i'll add

Reply:
I've added tooltips to the "Recreate columns" and the "Add column" buttons